test/users.js

```diff
@@ -57,3 +63,45 @@ describe('/users POST', () => {
         expect(res).to.be.json;
       }));
 });
+
+describe('/users/sessions POST', () => {
+  it('should successfully login', () =>
+    createUser().then(() =>
+      chai
+        .request(server)
+        .post('/users/sessions')
+        .send({ email: correctUser.email, password: correctUser.password })
+        .then(res => {
+          expect(res).to.have.status(200);
+          expect(res).to.be.json;
+          expect(res.body).to.have.property('token');
+        })
+    ));
+  it('should fail to log in with empty or null email/password', () =>
+    chai
+      .request(server)
+      .post('/users/sessions')
+      .send({ email: '', password: null })
+      .then(res => {
+        expect(res).to.have.status(400);
+        expect(res).to.be.json;
+      }));
+  it('should fail to log in with invalid email', () =>
+    chai
+      .request(server)
+      .post('/users/sessions')
+      .send({ email: 'invalid@wolox.com.ar', password: correctUser.password })
+      .then(res => {
+        expect(res).to.have.status(401);
+        expect(res).to.be.json;
+      }));
+  it('should fail to log in with valid email but invalid password', () =>
+    chai
+      .request(server)
+      .post('/users/sessions')
+      .send({ email: correctUser.email, password: '01928374' })
+      .then(res => {
+        expect(res).to.have.status(401);
+        expect(res).to.be.json;
+      }));
+});
```

Comment:
this works because you put all `it` together, you should call the `createUser` in a beforeEach